src/components/api/codenvy-user.factory.js

```diff
@@ -58,9 +58,6 @@ export class CodenvyUser {
     // user roles
     this.isUserInRoleMap = new Map();
 
-    // fetch the user when we're initialized
```

Comment:
This operation and some other(IsAdmin ...) should be only in codenvy API. I am sure. But we need more time for this fix.  It needs a separate issue.